### ISSUE TITLE
Midterm posted

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci143#303]
The midterm is posted at: <https://github.com/mikeizbicki/pagila-midterm>.  Good luck :)

**Comment by mikeizbicki** [comment]
**tl;dr** The corrected midterm has now been uploaded.  (If you have already cloned the repo, the easiest way to get to the corrected version is to delete your clone and reclone.)  Good luck!

**detailed postmortem:** In class I pulled up the pagila-midterm repo and looked at the `sql` folder.  I saw sql commands already entered in the files, and assumed I had accidentally pushed the solutions instead of just the problems.  Actually, these were the questions/answers to last year's midterm instead of this year's.  Every year, I rewrite the midterm using the same repo but different branches.  After rewriting the sql problems, I committed the changes to the wrong branch, and so when I pushed the `2023spring` branch to github, they did not contain the updated sql problems; they still contained the problems/solutions to last year's midterm.  I fixed the problem by merging the changes into the correct branch and uploading the correct branch to github.

**Comment by mikeizbicki** [comment]
The following clarification questions were asked about the midterm, and I'm answering them here for everyone:

> For question 1:
> Is it asking to exclude only the uppercase letter ‘F’ or lowercase as well?
>
> There is also a “District” column, but that is not included in the specific list of address parts we need to include. Should we assume that to be included as well or stick to the list? Seems like a superstitious person would not want an upper or lower case F at any part including districts.

The solution should account for both upper and lowercase 'F' (i.e. both `f` and `F`, and notice the formatting difference).  Since the original question didn't ask for the "District" column, there's no need to include it, but there's no penalty for including it either.

> Question 2:
> It is asking for simply the country and not the number of customers as well, right?

Correct.  Just the country should be output

> Question 3:
> It is asking for actors that have "appeared in movies in the "Children" category". Do we interpret this as the actor having appeared in at least 1 Children's movie, or at least 2 Children's movies since there is a plural? 

Actors appearing in at least 1 children's movie.
